PhysicsTools/PatUtils/plugins/ShiftedParticleProducer.cc

```diff
@@ -0,0 +1,77 @@
+#include "PhysicsTools/PatUtils/plugins/ShiftedParticleProducer.h"
```

Comment:
this file looks like a small variation from PhysicsTools/PatUtils/interface/ShiftedParticleProducerT.h
To reduce code proliferation (with epsilon different names), use ShiftedParticleProducerT, possibly modified to fit the extra uncertainty binning functionality